File: C/24BCSH93/1st Semester/test/question7.c
#include <stdio.h>

int main() {
	int size, i, even = 0, odd = 0;
	printf("Enter the size of the array: ");
	scanf("%d", &size);
	int array[size];
	printf("Enter the array elements: ");

	for (i = 0; i < size; i++) {
		scanf("%d", &array[i]);
	}	
	for (i = 0; i < size; i++) {
		if (array[i] % 2 == 0) {
			even++;
		}
		else {
			odd++;
		}
	}
	printf("No. of even numbers in the array is %d\n", even);
	printf("No. of odd numbers in the array is %d\n", odd);
	return 0;
}

File: Java/24BCSH93/Sec-A/assignment2/question7.java
/*  Q7. Given a sorted array, remove the duplicate elements from the array.
        Input: size of the array(n) and n numbers (the elements of the array).
        You are supposed to output the size of the new array followed by the elements of the new array after removing duplicates.
*/

import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the size of the array: ");
        int n = sc.nextInt();
        int[] array = new int[n];
        System.out.print("Enter the array elements: ");

        for (int i = 0; i < n; i++) {
            array[i] = sc.nextInt();
        }
        int i = 0;
        for (int j = 0; j < n; j++) {
            if (array[i] != array[j]) {
                array[++i] = array[j];
            }
        }
        int newSize = i + 1;
        System.out.print("The array elements after removing duplicates are: ");

        for (int j = 0; j < newSize; j++) {
            System.out.print(array[j] + " ");
        }
        System.out.println();
    }
}
